language/id.yml

```diff
@@ -259,10 +259,10 @@ muting-help: |
   × /mute <Melalui id pengguna atau username>: Membisukan seseorang di supergroup.
   × /mute <Melalui id pengguna atau username> X<m|h|d>: Membisukan sementara seseorang untuk X time.
   × /unmute <Melalui id pengguna atau username>: Tidak membisukan seorang pengguna yang dibisukan.\n
-  Contoh untuk membisukan sementara seseorang :
+  Contoh untuk membisukan sementara seseorang:
   m = Menit, h = Jam and d = Hari.
   ``/mute @username 2h``; Pengguna ini akan dibisukan selama 2 jam.
-invalid-time-flag: Tidak sah atau benar tipe waktu. Tolong gunakan m,h, or d.
+invalid-time-flag: Tidak sah atau tidak benar tipe waktu. Tolong gunakan m,h, atau d.
```

Comment:
I don't thing this is right.
`Tipe waktu tidak sah .... `?

Reply:
Ok sir. Sorry for late reply. I forgot to log in my email to my phone after flash the device hahaha.